platform_it/src/test/java/com/splicemachine/test/SpliceTestPlatformConfig.java

```diff
@@ -321,8 +321,8 @@ public static Configuration create(String hbaseRootDirUri,
         // Replication
         //
         config.setBoolean("replication.source.eof.autorecovery", true);
-        config.set("hbase.replication.source.service", "com.splicemachine.replication.SpliceReplication");
-        config.set("hbase.replication.sink.service", "com.splicemachine.replication.SpliceReplication");
+        //config.set("hbase.replication.source.service", "com.splicemachine.replication.SpliceReplication");
+        //config.set("hbase.replication.sink.service", "com.splicemachine.replication.SpliceReplication");
```

Comment:
Why is that necessary?

Reply:
Comment them out so that splice can be started w/o building ee branch, because SpliceReplication is only available on ee branch. Without commenting them out, the following command will fail
./start-splice-cluster -pcdh6.3.0

You have to always build with ee branch
start-splice-cluster -pcdh6.3.0,ee